packages/react-devtools-extensions/src/background.js

```diff
@@ -123,9 +166,6 @@ chrome.runtime.onMessage.addListener((request, sender) => {
     // This is sent from the hook content script.
     // It tells us a renderer has attached.
     if (request.hasDetectedReact) {
-      // We use browserAction instead of pageAction because this lets us
-      // display a custom default popup when React is *not* detected.
-      // It is specified in the manifest.
```

Comment:
Sanity check: Did we test to make sure this works in manifest v3? (Assuming we did but leaving it here just in case!)

Reply:
Yes. I did check that